docs/changelog/1_0_a6.rst

```diff
@@ -0,0 +1,63 @@
+===========
+1.0 Alpha 6
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 6 "Wolf".
+
+
+EdgeQL
+======
+
+* Introduce new enum definition syntax (:eql:gh:`#1843`).
+
+  Instead of using strings, the enums are now defined using identifiers:
+
+  .. code-block:: edgeql
+
+    CREATE SCALAR TYPE schema::Cardinality
+        EXTENDING enum<ONE, MANY>;
+
+* Rename built-in schema enums to use CamelCase names (:eql:gh:`#1843`).
+* Rename string handling functions to use ``start`` and ``end`` in
+  their name (:eql:gh:`#1752`).
```

Comment:
Should we list functions? There are only 4 of them as far as I know.

Reply:
How about calling it upper camel case? Probably not super important but I remember at least two people on Twitch who were annoyed at how Rust's structs, enums etc. were written in what it called camel case which they insisted was pascal case. Eventually the Rust docs ended up calling it upper camel case and that's what the error message gives now:

>warning: type `someStruct` should have an upper camel case name

Doing that now might avoid some pedantry later on.

Reply:
Yes, upper camel case sounds better than pascal case (nobody remember Pascal these days :) )
*(But this probably should be in a different comment)*

Reply:
Agree with @tailhook; please least all functions (old, and their new equivs) in a table.

Reply:
I've added the list of affected functions.